lib/streamaudio.h

```diff
@@ -143,9 +143,15 @@
 #include "ssl_lib.h"
 #include "TaskSystem.h"
 
+extern "C" {
+#include "libacm.h"
+}
```

Comment:
Headers are supposed to take care of themselves. Only write `#include "libacm.h"` here.
In libacm.h add this before declarations
```
#ifdef __cplusplus
extern "C" {
#endif

```
and then this after the declarations
```
#ifdef __cplusplus
}
#endif

```

Reply:
Fair enough. I didn't want to modify the code from the other library so we could drop in a new version.